languages/python/openai_rllib/README.md

```diff
@@ -0,0 +1,85 @@
+# Create Anaconda environment
+
+This repo is a tutorial for installing and using OpenAI Gym on Eagle, as well as running experiments on single/multiple cores and nodes.
+
+Below are the basic steps for creating a dedicated Anaconda environment that you will use for all your experiments. Note that this environment will contain only some basic packages, but you can always installmore packages depending on your needs (please see at the bottom of this README file).
+
+If you have any questions, you can email us in the following address:
+* Erotokritos Skordilis: Erotokritos.Skordilis@nrel.gov
+
+## 1<sup>st</sup> step: Logging in on Eagle
+
+Login on Eagle with:
+```
+ssh eagle
+```
+or
+```
+ssh <username>@eagle.hpc.nrel.gov
+```
+
+## 2<sup>nd</sup> step: Set up Anaconda environment
+
+Use the `env_example.yml` file to create the new Anaconda environment. You can install the environment to the directory of your choosing. There are three main directories on Eagle where you can install the new environment, namely `/home`, `/scratch`, and `/projects`. Depending on your needs, you have to choose one of these three. For more information regarding installing your new environment and the different Eagle directories, please see [here](https://nrel.github.io/HPC/languages/python/NREL_python.html) and [here](https://nrel.github.io/HPC/languages/python/conda.html).
```

Comment:
It is generally best to create [links that are descriptive]() instead of linking the "[here]()" (blank link examples).

I'll be sure to add this to the style guide!

Reply:
Good idea, thank you! Changed